src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -227,7 +229,7 @@ def __init__(
             num_planes=self._num_planes,
             num_acquisition_cycles=num_acquisition_cycles,
             num_frames_per_slice=self._frames_per_slice,
-            num_flyback_frames=self.num_flyback_frames,
+            num_flyback_frames_per_channel=self.num_flyback_frames_per_channel,
```

Comment:
Nit: I feel like per_channel is kinda unnecessary. `frames_per_slice` is understood to be per-channel as is `num_planes`.

Reply:
Mmm maybe you are right. But I was really confused about it so I wanted to make it clear.